### ISSUE TITLE
Notification

### ISSUE THREAD

**Issue by AshwaniGautam** [nishanthvijayan/CoderCalendar#2]
It would be more helpful if it display a notification icon, to draw attention, i think so.

**Comment by nishanthvijayan** [comment]
Notification icon? I don't think I understand what you're trying to say.Could you be more specific?

**Comment by AshwaniGautam** [comment]
Ok, as a user I sometimes get busy with other things on internet(Read fb,youtube,academic stuff) and thus I might sometime miss to click the add-on. Have you seen Adblock add-on, it displays a small red circle at the bottom of its logo(or whatever) representing the number of add blocked by it, that red mark draws attention even if its not meant to. As you have implemented a check box style for selecting preferred website for contest lists, on similar line you can introduce a check box style for selecting famous contest held every week, month,year on website, For instance list can include


*Codechef Monthly challenge

*Codechef Monthly Cook-off

*Codechef Lunchtime

*Codeforces Round #__

*TopCoder SRM #__

etc….

If a user select first, second and third then a notification mark would be displayed showing that one of users interest challenge is inching closer,  though when he would click the add-on he would be able to see all challenges for his selected website as is right now,  this feature would

drag user to frequently open the add-on showing him/her the list. If its not clear, I would try to implement a dummy …😊  



From: Nishanth Vijayan
Sent: ‎Saturday‎, ‎02‎ ‎May‎ ‎2015 ‎00‎:‎46
To: nishanthvijayan/CoderCalendar
Cc: Ashwani Gautam





Notification icon? I don't think I understand what you're trying to say.Could you be more specific?

**Comment by nishanthvijayan** [comment]
@jainendra is implementing something similar in #26 
In #26 instead of displaying a number representing selected contests, the number of new contests will be displayed.I think this is a better idea.
Maybe we could think of integrating your idea into this somehow, but it's not a priority.
